### ISSUE TITLE
Loft Splines not observing vertex hooks

### ISSUE THREAD

**Issue by MasterJames** [JacquesLucke/animation_nodes#863]
I first asked this with Blender bug tracker and got a "please ask the addon developer instead" request [+ closed].
Then I posted on Stack Exchange (prior to listing here) and got a super great helpful response there, which re-quoted an AN dev saying essentially Blender does not....

> "allow scripts to get the result of curve modifiers in a way that is suitable for AN."

I've therefor reopened the complaint there with a simple demo file attached and a link to the comment on stack exchange requesting there be some further hope that this could be resolved somehow one day, or not.
This is the SX link...
https://blender.stackexchange.com/questions/100628/an-loft-splines-not-observing-vertex-hooks

And here the Bug Tracker at Blender...
https://developer.blender.org/T53998

Maybe you too can support the cause with comments/input or maybe attempt to "rephrase" the bug in a way that makes more sense to them?
Possibly there are more articles and issues raised that could also be linked in to help understand and track further any possible progress on this issue. [double thanks]

**Comment by stansu** [comment]
![image](https://user-images.githubusercontent.com/9460236/37185008-2f87ec30-2379-11e8-9d15-9f5adb2aff01.png)
![image](https://user-images.githubusercontent.com/9460236/37185050-66548c8c-2379-11e8-9c34-a12acee52027.png)
![image](https://user-images.githubusercontent.com/9460236/37185062-739a8e82-2379-11e8-8915-5b009ef2ce62.png)

**Comment by MasterJames** [comment]
Looks good I tried it, but see nothing. One difference was the "Create Spline List" has a "List (2)" second default element that has an empty picker. I guess this example is from a different version? I'm using v 2.79

The BezierCurve object is a single object with a curve that has extra points added and has been duplicated with some variations in X,Y so they are only stacked in Z without variation in Z per similar curve.

I'll look at it more later but any words to explain why this latest example I'm thankful for is doing nothing like the result shown would be appreciated, thanks again for the demonstration @stansu .

**Comment by stansu** [comment]
[loft_hook_spline.blend.zip](https://github.com/JacquesLucke/animation_nodes/files/1796180/loft_hook_spline.blend.zip)

**Comment by MasterJames** [comment]
Yup that works, it was multiple splines (bezier curves) I couldn't see in the original graphic as it was chopped off. I totally would(a shoulda) have tried that next.
This example has "Animate always off", which seemed like it didn't work at first and the extra node locked it back to the same position. I was able to animate the empty and see real-time updates after deleting the extra dangling animation node that was returning the value to a constant but was also another good lesson in another option to control it. [These are just notes to someone else exploring the example given.]
Anyway thanks for the example!
I'll need to follow up on what I was doing before that others said wasn't possible, and how to learn from this a better way to understand that and AN processes in general. I'll report back if something becomes clearer to me later that is worth adding. Closing for now and again thanks. 
謝謝
